test/suite_test.go

```diff
@@ -66,6 +68,16 @@ func (e *e2e) SetupSuite() {
 		)
 		e.Nil(os.Chmod(e.kindPath, 0o755))
 	}
+
+	e.kubectlPath = filepath.Join(buildDir, "kubectl")
+	if !util.Exists(e.kubectlPath) {
+		e.run(
+			"curl", "-o", e.kubectlPath,
+			"https://storage.googleapis.com/kubernetes-release/release/"+
+				kubectlVersion+"/bin/linux/amd64/kubectl",
+		)
+		e.Nil(os.Chmod(e.kubectlPath, 0o755))
```

Comment:
Given the nature of this setup I would probably keep the ACL focused on the current user.
```suggestion
		e.Nil(os.Chmod(e.kubectlPath, 0o700))
```

Reply:
Fixed :+1: